lib/pages/timetable/view/timetable_page.dart

```diff
@@ -66,7 +66,22 @@ class _TimetablePageState extends State<TimetablePage> {
       needsToBeAuthenticated: true,
       leading: AppScaffoldAction(
         icon: Icons.today_outlined,
-        onPressed: () => _controller.animateToToday(),
+        onPressed: () =>
+            !_controller.currentlyVisibleDates.contains(LocalDate.today())
+                ? _controller.animateToToday()
+                : ScaffoldMessenger.of(context).showSnackBar(
+                    const SnackBar(
+                      backgroundColor: Color(0xFF43ACCD),
+                      content: Text(
+                        'You\'re already seeing the current week!',
```

Comment:
The app works in both Romanian and English, so all user-facing strings have to be localized. Please read this for info about how we do localization: https://github.com/student-hub/acs-upb-mobile/blob/master/CONTRIBUTING.md#on-device